orchagent/fdborch.cpp

```diff
@@ -483,6 +475,11 @@ bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name, const
     attr.value.s32 = SAI_PACKET_ACTION_FORWARD;
     attrs.push_back(attr);
 
+    if (m_entries.count(entry) != 0) // we already have such entries
+    {
+        removeFdbEntry(entry);
```

Comment:
What if removeFdbEntry failed to remove the entry from SAI? There is a 'FIXME' in that code path. What will happen for the subsequent create_fdb_entry() call into SAI?

Look like the code flow will stop retry below by returning false at the 'FIXME' line?

Reply:
Yes,  there is a 'FIXME' in the removeFdbEntry() code path. When sai_fdb_api->remove_fdb_entry() found the entry to be removed is not exist in SAI, it must return SAI_STATUS_SUCCESS. In this scenario, when one entry is removed in ASIC, and there is a time difference to notify the event to ASIC_DB.